src/main/java/com/icthh/xm/uaa/config/UaaAccessTokenConverterConfiguration.java

```diff
@@ -124,4 +130,18 @@ public JwtTokenStore tokenStore(JwtAccessTokenConverter jwtAccessTokenConverter)
         return new JwtTokenStore(jwtAccessTokenConverter);
     }
 
+    private PrivateKey initPrivateKeyFromKeystore(InputStream stream) throws KeyStoreException, UnrecoverableKeyException,
+        NoSuchAlgorithmException, IOException, CertificateException {
+        KeyStore store = KeyStore.getInstance(Constants.KEYSTORE_TYPE);
+        store.load(stream, applicationProperties.getKeystorePassword().toCharArray());
+        return (PrivateKey) store.getKey("selfsigned",
```

Comment:
I think it worth to extract key name inside keystore to properties with `swlfsigned` as default value

Reply:
parameter name like `kaystore-key-name`